src/mmw/js/src/data_catalog/templates/resultDetailsCuahsi.html

```diff
@@ -6,33 +6,43 @@
         <h2>
             Site: {{ location }} {{ title }}
         </h2>
-        <p>
-            Source: {{ service_org }} {{ service_title }}
-            <a data-toggle="popover" tabindex="0"
-               data-html="true" data-container="body" role="button"
-               data-content="{{ description }}"
-               data-template="<div class='popover'><div class='pull-right' id='popover-close-button' onclick='closePopover()'><i class='fa fa-times' /></div><div class='popover-content'></div><div class='arrow'></div></div>">
-                <i class="fa fa-info-circle black"></i>
-            </a>
-        </p>
+        <table class="table-data-detail">
+            <tr>
+                <th>Source</th>
+                <td>{{ service_org }} {{ service_title }}
```

Comment:
I think in most real world use, the `service_org` and `service_title` are not going to fit in one line:

![image](https://user-images.githubusercontent.com/1430060/30652389-44c8d45a-9df6-11e7-96f7-13455ef6ce6d.png)

Let's add a `<br />` between them so it looks more even:

![image](https://user-images.githubusercontent.com/1430060/30652470-798b71e8-9df6-11e7-86f5-334a7b9ff3e4.png)

Also, consider adding some left padding to the right column, because natural table resize can being some left labels visually too close to the values on the right.